ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/JsonRpcConfiguration.java

```diff
@@ -41,6 +41,7 @@
   private String host;
   private List<String> corsAllowedDomains = Collections.emptyList();
   private List<String> rpcApis;
+  private List<String> noAtuhRpcApis = Collections.emptyList();
```

Comment:
typo in member

Reply:
[db39e60](https://github.com/hyperledger/besu/pull/3382/commits/db39e60b1fc916904e18b67979b55bb7f9977942)